sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -177,6 +179,7 @@ public boolean deactivateUserByLogin(DbSession dbSession, String login) {
     mapper.deleteUserMeasureFilterFavourites(dto.getId());
     mapper.deleteUserProperties(dto.getId());
     mapper.deleteUserRoles(dto.getId());
+    mapper.deletePropertiesMatchingLogin(asList(DEFAULT_ISSUE_ASSIGNEE), dto.getLogin());
```

Comment:
First point : done in deleteUserProperties(), here we need to remove a property value that match a user login.
Second point : I think the call is to have specific SQL queries for the purge of a user.